changelogs/fragments/setup.ps1-parity-with-linux-regarding-missing-local-facts-path.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
```

Comment:
You can definitely argue whether this is a bug fix or not but I would say this is more a change as I would not be able to backport this to earlier releases. I think we should change this to `minor_changes` and add this change in behaviour to the porting guide for 2.10 https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.10.rst.

Reply:
add it in this pr or a different one?